Sources/Common/Util/KeyValueStorage.swift

```diff
@@ -29,7 +29,7 @@ public protocol SandboxedSiteIdKeyValueStorage {
  Uses UserDefaults to store data in key/value pairs.
  */
 // sourcery: InjectRegisterShared = "SandboxedSiteIdKeyValueStorage"
-public class UserDefaultsSandboxedSiteIdKeyValueStorage: SandboxedSiteIdKeyValueStorage {
+public class UserDefaultsSandboxedSiteIdKVStore: SandboxedSiteIdKeyValueStorage {
```

Comment:
Any specific reason to change this class name? I think the name was exceeding the length, correct? 🤔

Reply:
Yup name of the class, open to suggestions was struggling alot with it 😅